packages/elements/src/components/ino-input/ino-input.scss

```diff
@@ -199,6 +199,10 @@ ino-input .mdc-text-field--outlined {
     @include textfield.focused-outline-color(theme.$error);
   }
 
+  & .mdc-floating-label:not(.mdc-floating-label--float-above) {
```

Comment:
Does not seem to be fixed quite yet:

![image](https://github.com/inovex/elements/assets/22963121/abca8f4e-730c-4375-82c3-ee5bca47e6d4)

These changes seem to have no effect when the label is floating above.